core/src/main/java/io/kestra/core/models/namespaces/DisabledInterface.java

```diff
@@ -0,0 +1,5 @@
+package io.kestra.core.models.namespaces;
+
+public interface DisabledInterface {
+    Boolean getDisabled();
```

Comment:
`isDisabled` so that we keep the same naming convention than the `DeletedInterface`, and it's a more common naming rule in java.